### ISSUE TITLE
Hide fields based on option doesnt work

### ISSUE THREAD

**Issue by soyoh** [bmichotte/ProMotion-XLForm#18]
Hi,
I was testing the "hide" functionality and it doesn't work for options. but it works when is a switch.

i tried the code from the test file:

``` ruby
{
  title: 'Options',
  name: 'options',
  type: :selector_push,
  appearance: {
    font: UIFont.fontWithName('Helvetica Neue', size: 15.0),
    detail_font: UIFont.fontWithName('Helvetica Neue', size: 12.0),
    color: UIColor.greenColor,
    detail_color: UIColor.blueColor,
    background_color: UIColor.grayColor
  },
  options: {
    "value_1" => "Value 1",
    "value_2" => "Value 2",
    "value_3" => "Value 3",
    "value_4" => "Value 4",
  },
  value: "value_1",
  on_change: -> (old_value, new_value) {
    mp old_value: old_value,
       new_value: new_value
  }
},
{
  title: 'Value 1 ?',
  name: :show_me_selector,
  type: :text,
  hidden: { name: :options, is: :equal, value: 'value_1', options: true },
  value: 'is selected !'
},
```

from https://github.com/bmichotte/ProMotion-XLForm/blob/master/app/screens/test_form_screen.rb#L71 and https://github.com/bmichotte/ProMotion-XLForm/blob/master/app/screens/test_form_screen.rb#L242

it should only display the "Value 1? is selected!" when is that one selected, instead is not changing anything.

is this a known issue or i'm doing something wrong??
thank you!!!

**Comment by bmichotte** [comment]
Oops, seems I did not see this one. 

It has been corrected by #13, but I don't have bump a new release since the PR. Could you try with master instead of 0.0.10 ? (and ofc, use the code from the PR)

**Comment by soyoh** [comment]
awesome! thanks @bmichotte
